app/src/main/java/org/wikiedufoundation/wikiedudashboard/ui/home/HomeActivity.kt

```diff
@@ -64,6 +66,10 @@ class HomeActivity : AppCompatActivity() {
             searchView.queryHint = "Search"
             searchView.isIconified = false
 
+        val txtSearch = searchView.findViewById(androidx.appcompat.R.id.search_src_text) as EditText
```

Comment:
Can we do the same in XML ?

Reply:
I don't think so because SearchView is implemented using the `actionViewClass` attribute in the `menu_explore.xml` file.

That is why
```
searchView.queryHint = "Search"
searchView.isIconified = false
```
is also being done in `HomeActivity.kt`.

Reply:
Got it. Thanks for clarification.